data/White_2020/metadata.yml

```diff
@@ -422,12 +421,36 @@ substitutions:
 - trait_name: plant_tolerance_water_logged_soils
   find: '<1_month'
   replace: less_than_1_month
+- trait_name: plant_tolerance_water_logged_soils
+  find: '<1_month 1-6_months'
```

Comment:
are these correct values? Here and below?

Reply:
Yes... The first problem was that Protege couldn't read in "<" or ">", so I had to write those out. But I hadn't picked up that in White_2020 there are lots of species that have been scored as falling into multiple categories. A bigger change -- to be done in consultation with others -- would be to change them all into bins, and then instead of being scored as fitting into two narrow bins, there would be a single broader one.